### DISCUSSION TITLE
Separate driver for MariaDB?

### DISCUSSION THREAD

**Discussion by louis77** [go-jet/jet#396] [Ideas]
MariaDB is diverging significantly from MySQL. For example, it supports the RETURNING clause, MySQL does not. Wouldn't it make sense to clone the `mysql` package and maintain a separate version as `mariadb`, so we can add/modify it to accommodate for all the new stuff that MariaDB supports?

**Comment by houtn11** [comment]
That might be too excessive for just one clause. What else does MariaDB now have?
